app/src/main/java/io/gnosis/safe/ui/transactions/TransactionListViewModel.kt

```diff
@@ -226,7 +249,8 @@ class TransactionListViewModel
     private fun Transaction.queuedCustomTransaction(
         txInfo: TransactionInfo.Custom,
         safeAddress: Solidity.Address,
-        awaitingYourConfirmation: Boolean
+        awaitingYourConfirmation: Boolean,
+        isConflict: Boolean
```

Comment:
Were the unit tests not affected by the addition of this flag?

Reply:
I added a check for empty nonces in conflicting transfers. See: 40368ea242a1838cae95c03aaf07579a479e8586